partcad/src/partcad/__init__.py

```
@@ -23,18 +28,45 @@
 
 __all__ = [
+    "Assembly",
+    "Context",
+    "Location",
+    "Part",
+    "Project",
+    "ProviderCart",
+    "ProviderRequestQuote",
+    "Shape",
+    "UserConfig",
+    "assembly",
```

Ruff (0.8.2)

`assembly` may be undefined, or defined from star imports

(F405)